docs/community-ee/community-ee-release-process.md

```diff
@@ -11,68 +11,65 @@ The EE versioning convention is core tag plus patch number, for example:
     - EE with core `2.16.2` comes out -> `community-ee:2.16.2-1`
     - EE with core `2.16.3` comes out -> `community-ee:2.16.3-1`
 
-## Dependencies
-
-- Podman/Docker
-
 ## Credentials
 
-- Access to the https://github.com/ansible-community/images repository
-- Access to ghcr.io
+- Access to https://github.com/ansible-community/images repo.
+- Join [Release Management working Matrix room](https://app.element.io/?updated=1.11.38#/room/#release-management:ansible.com) and [Execution Envs group in Forum Group](https://forum.ansible.com/g/ExecutionEnvs).
+- Access to Ansible Release Management Group in Github.
+- Access to [eercheck](https://github.com/anweshadas/eercheck) repo.
```

Comment:
```suggestion
- Access to the [eercheck](https://github.com/anweshadas/eercheck) repository.
```

Reply:
If this repository is an essential part of the EE release workflow it needs to move into the `ansible-community` org. @gundalow FYI.

Reply:
Agreed.